zaza/openstack/charm_tests/vault/tests.py

```diff
@@ -63,8 +63,17 @@ def setUpClass(cls):
         if cls.vip_client:
             cls.clients.append(cls.vip_client)
         cls.vault_creds = vault_utils.get_credentials()
+
+        # This little dance is to ensure a correct init and unseal sequence,
+        # for the case of vault with the raft backend.
+        # It will also work fine in other cases.
```

Comment:
Note, that I'd add a note here that this will raise an AssertionError if the vault units don't unseal for the class.